SPECS/SymCrypt/SymCrypt.spec

```diff
@@ -1,8 +1,7 @@
-%define debug_package %{nil}
```

Comment:
I'm a little concerned that if you have not rebuilt SymCrypt-OpenSSL or OpenSSL packages which depend on this, you won't have tested that the produced libsymcrypt.so works with only the current %check in this spec.

When building modules in ADO we test the produced dynamic module with symcryptunittest, rather than just testing that symcryptunittest (with statically linked SymCrypt) is working as expected: https://github.com/microsoft/SymCrypt/blob/171f6973dab9b76f0dc61d966d3e977021325bc1/.pipelines/templates/build-linux.yml#L128-L135

This is independent of needing to do manual validation that the new debug symbols work as expected

Reply:
Does this mean anytime we update one of SymCrypt-OpenSSL, OpenSSL, or SymCrypt, we actually need to rebuild and redistribute all 3 in order for debug symbols to work?

Reply:
Sorry for the ambiguity here - I just meant that we should run the tests from SymCrypt-OpenSSL + OpenSSL spec files when we update this SymCrypt spec file, as the test section of this build recipe is not as in depth for how callers use the produced libsymcrypt.so.